types/mathjs/index.d.ts

```diff
@@ -2571,6 +2572,19 @@ declare namespace math {
          * ‘string’, ‘Array’, ‘Date’.
          */
         typeof(x: any): string;
+
+        /**
+         * Import functions from an object or a module
+         * @param object An object with functions to be imported.
+         * @param options An object with import options.
+         */
+        import(object: ImportObject | ImportObject[], options: ImportOptions): void;
+        // TypeScript Version: 2.4
+
+        /**
+         * It may not exists - this is _only_ for import functionality.
+         */
+        [key: string]: any;
```

Comment:
Personally, I think I would avoid this in favor of forcing consumers to manually merge new functions into the namespace.

For example:
```typescript
// imported_math_functions.ts
declare module 'mathjs' {
  namespace math {
    hello(a: number): number;
  }
}
```